router/pkg/pubsub/nats/nats.go

```diff
@@ -61,10 +90,14 @@ func (p *natsPubSub) Subscribe(ctx context.Context, event pubsub_datasource.Nats
 	)
 
 	if event.StreamConfiguration != nil {
-		consumer, err := p.js.CreateOrUpdateConsumer(ctx, event.StreamConfiguration.StreamName, jetstream.ConsumerConfig{
-			Durable:        event.StreamConfiguration.Consumer, // Durable consumers are not removed automatically regardless of the InactiveThreshold
+		consumerConfig := jetstream.ConsumerConfig{
+			Durable:        p.getDurableConsumerName(event.StreamConfiguration.Consumer, event.Subjects), // Durable consumers are not removed automatically regardless of the InactiveThreshold
```

Comment:
This comments seems wrong and needs to be updated

Reply:
nice catch, thanks!